project-base/config/services_test.yml

```diff
@@ -10,146 +10,49 @@ services:
         autoconfigure: true
         public: true
 
-    Shopsys\FrameworkBundle\Component\Localization\DateTimeFormatter:
-        factory: ['@Shopsys\FrameworkBundle\Model\Localization\CustomDateTimeFormatterFactory', create]
-
-    CommerceGuys\Intl\Currency\CurrencyRepositoryInterface:
-        class: Shopsys\FrameworkBundle\Model\Localization\IntlCurrencyRepository
-
     Shopsys\FrameworkBundle\Component\EntityExtension\EntityNameResolver:
         class: Tests\App\Functional\EntityExtension\OverwritableEntityNameResolver
         arguments:
             - "%shopsys.entity_extension.map%"
 
     Tests\App\Functional\EntityExtension\OverwritableEntityNameResolver: '@Shopsys\FrameworkBundle\Component\EntityExtension\EntityNameResolver'
 
-    joschi127_doctrine_entity_override.event_subscriber.load_orm_metadata:
+    Tests\App\Functional\EntityExtension\OverwritableLoadORMMetadataSubscriber:
         class: Tests\App\Functional\EntityExtension\OverwritableLoadORMMetadataSubscriber
```

Comment:
This is not needed when service name is same as class

Reply:
:+1: 917e93545b41c043938afe2487f534d0ed8fa5d6